virtcontainers/kata_proxy.go

```diff
@@ -51,6 +51,8 @@ func (p *kataProxy) start(params proxyParams) (int, string, error) {
 		return -1, "", err
 	}
 
+	go cmd.Wait()
```

Comment:
but.. but.. someone has to wait for this new routine, :confused: :question:

Reply:
> but.. but.. someone has to wait for this new routine, 😕 ❓

Goroutine is a thread.  So when kata-runtime quit, it will quit with it.